game/src/main/java/org/apollo/game/message/impl/SendPlaySoundMessage.java

```diff
@@ -0,0 +1,67 @@
+package org.apollo.game.message.impl;
+
+import org.apollo.net.message.Message;
+
+/**
+ * A {@link Message} sent to the client to play a sound.
+ *
+ * @author tlf30
+ */
+public final class SendPlaySoundMessage extends Message {
+
+	/**
+	 * The id of the sound to play.
+	 */
+	private final int id;
+
+	/**
+	 * The delay that the client should use before playing the sound.
+	 */
+	private final int delay;
+
+	/**
+	 * The type of sound to play.
+	 */
+	private final int type;
+
+	/**
+	 * Creates a new send play sound message.
+	 *
+	 * @param id The id of the sound to play.
+	 * @param type The type of the sound to play.
```

Comment:
Do we know what the different types are?

Reply:
So far all the sounds I have used are using type 0